fuse/data/ops/ops_common.py

```diff
@@ -381,6 +381,22 @@ def __call__(self, sample_dict: NDict, keep_keypaths: List[str]) -> Union[None,
         return sample_dict
 
 
+class OpDeleteKeypaths(OpBase):
+    """
+    Use this op to remove keypaths from the sample
+    A case where this is useful is if you want to limit the amount of data that gets transferred by multiprocessing by DataLoader workers.
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
I think you can delete the `__init__` function and the logic remain the same. If so, I guess it's preferred. what do you think?